umbraco/Umbraco-CMS
# Granting access to the content dashboard for all user groups

## Details 
Adds the `AllowContentDashboardAccessToAllUsers` appSetting in the Web.config file, so the intro dashboard ("Getting Started") is shown to all user groups, not just admins. The change will affect all new projects. In the case that the config setting is not present the old access rules will apply, therefore not changing any existing behaviour.

Additionally, more information is reported from an existing dashboard - an admin parameter is added by the end.

## Test
- Remove the **AllowContentDashboardAccessToAllUsers** entry from the Web.config or change its value to `false` and verify that only users in the admin group have access to the Content dashboard;
- Add it back or set it to `true` and create a few new users (ex: belonging to the Editors or Writers group) and check that the Content dashboard also appears (in addition to Redirect URL Management which was already there);

-----
**Note:**
SensitiveData and Translators users have different UIs so the dashboard is not present at first glance, however, they have access to it!
